webui: better styling and interface

The initial HTML setup for our webui was quickly tossed together, but we should offer a prettier interface that's more intuitive for users. 

![Screen Shot 2021-06-30 at 3 49 14 PM](https://user-images.githubusercontent.com/120951/124040940-c2c88e00-d9ba-11eb-95ce-da8f614334bc.png)

@adamdecaf a template or asset directory would be lovely as opposed to modifying those prints.

I'm running `make run` from the project root and I get:
```
 ach-web-viewer git:(master) make run
pkger -include /migrations -include /configs/config.default.yml
go mod vendor
go build -mod=vendor -ldflags "-X github.com/moov-io/ach-web-viewer.Version=v0.2.1" -o bin/ach-web-viewer github.com/moov-io/ach-web-viewer/cmd/ach-web-viewer
./bin/ach-web-viewer
ts=2021-07-01T23:07:32Z msg="loading config file" level=info version=v0.2.1 app=ach-web-viewer component=Service file=/configs/config.default.yml
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x1658eb3]

goroutine 1 [running]:
github.com/moov-io/base/admin.NewServer(0xc0003a1157, 0x5, 0xc0003c6dc0)
	/Users/joshsadler/moov/ach-web-viewer/vendor/github.com/moov-io/base/admin/admin.go:43 +0xb3
github.com/moov-io/ach-web-viewer/pkg/service.bootAdminServer(0xc0003fe4e0, 0x1b4c528, 0xc0003887e0, 0xc0003a1157, 0x5, 0x0, 0x0, 0x0)
	/Users/joshsadler/moov/ach-web-viewer/pkg/service/server.go:65 +0x39
github.com/moov-io/ach-web-viewer/pkg/service.(*Environment).RunServers(0xc00063ff28, 0xc0003fe4e0, 0x0)
	/Users/joshsadler/moov/ach-web-viewer/pkg/service/server.go:22 +0x73
main.main()
	/Users/joshsadler/moov/ach-web-viewer/cmd/ach-web-viewer/main.go:36 +0x325
make: *** [run] Error 2
```

--- a/webui/file.html.tpl
+++ b/webui/file.html.tpl
@@ -1,19 +1,27 @@
 <!doctype html>
 <html>
   <head>
-    <title>{{ .Filename }} | ACH Viewer</title>
+    <title>{{ .Filename }} | ACH viewer</title>
     <link rel="stylesheet" href="{{ .BaseURL }}style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;500;700&family=Manrope:wght@400;500;700&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,viewport-fit=cover">
+    <meta name="theme-color" media="(prefers-color-scheme: light)" content="#fefefe">
+    <meta name="theme-color" media="(prefers-color-scheme: dark)" content="#202020">
   </head>
   <body>
-    <a href="{{ .BaseURL }}">Back</a>
-    <br />
-    <pre>{{ .Contents }}</pre>
-    <br />
-    {{ if eq .Valid nil }}
-    Valid: true
-    {{ else }}
-    <strong>Validation Error</strong>:
-    <br /><pre>{{ .Valid }}</pre>
+    <header>
+      <a href="{{ .BaseURL }}"><- Back</a>
+      <h1>{{ .Filename }}</h1>
+    </header>
+    {{ if ne .Valid nil }}
+      <div class="error">
+        {{ .Valid }}
+      </div>
     {{ end }}
+    <main>
+      <pre>{{ .Contents }}</pre>
+    </main>
   </body>
 </html>

--- a/webui/index.html.tpl
+++ b/webui/index.html.tpl
@@ -1,16 +1,29 @@
 <!doctype html>
 <html>
   <head>
-    <title>ACH File Viewer</title>
+    <title>ACH file viewer</title>
     <link rel="stylesheet" href="{{ .BaseURL }}style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;500;700&family=Manrope:wght@400;500;700&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,viewport-fit=cover">
+    <meta name="theme-color" media="(prefers-color-scheme: light)" content="#fefefe">
+    <meta name="theme-color" media="(prefers-color-scheme: dark)" content="#202020">
   </head>
   <body>
-    {{ range $source := .Sources }}
-    <strong>{{ $source.ID }}</strong> ({{ $source.Type }})
-    {{ range $file := $source.Files }}
-    <br /><a href="{{ $file.Path }}">{{ $file.Filename }}</a>
-    {{ end }}
-    <br /></br />
-    {{ end }}
+    <header>
+      <h1>ACH file viewer</h1>
+    </header>
+    <main class="list">
+      {{ range $source := .Sources }}
+        <div class="source"><strong>{{ $source.ID }}</strong> ({{ $source.Type }})</div>
+        {{ range $file := $source.Files }}
+          <a href="{{ $file.Path }}">
+            <svg class="icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="m18 2h-12a2 2 0 0 0 -2 2v16a2 2 0 0 0 2 2h6.76a3 3 0 0 0 2.12-.88l4.24-4.24a3 3 0 0 0 .88-2.12v-10.76a2 2 0 0 0 -2-2zm0 12h-5a1 1 0 0 0 -1 1v5h-6v-16h12zm-9.5-6h7a.5.5 0 0 0 .5-.5v-1a.5.5 0 0 0 -.5-.5h-7a.5.5 0 0 0 -.5.5v1a.5.5 0 0 0 .5.5zm0 4h4a.5.5 0 0 0 .5-.5v-1a.5.5 0 0 0 -.5-.5h-4a.5.5 0 0 0 -.5.5v1a.5.5 0 0 0 .5.5z"></path></svg>
+            {{ $file.Filename }}
+          </a>
+        {{ end }}
+      {{ end }}
+    </main>
   </body>
 </html>

--- a/webui/style.css
+++ b/webui/style.css
@@ -1,1 +1,116 @@
 /* CSS for ach-web-viewer */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+html {
+  --background-color: #fafafa;
+  --header-background: rgba(255,255,255,.8);
+  --title-color: #101010;
+  --text-color: #333;
+  --accent-color: #219653;
+  --border-color: #f3f2f1;
+  font-family: 'Manrope', system-ui, -apple-system, BlinkMacSystemFont, sans-serif;
+  font-size: 16px;
+}
+@media (prefers-color-scheme: dark) {
+  html {
+    --background-color: #090909;
+    --header-background: rgba(37, 37, 37, 0.8);
+    --title-color: #fafafa;
+    --text-color: #bdbdbd;
+    --accent-color: #6fcf97;
+    --border-color: #333;
+  }
+}
+body {
+  margin: 0;
+  font-size: .875rem;
+  line-height: 1.5;
+  background-color: var(--background-color);
+  color: var(--text-color);
+}
+a {
+  color: var(--accent-color);
+  text-decoration: none;
+  font-weight: 500;
+}
+pre {
+  font-family: 'JetBrains Mono', monospace;
+}
+
+header {
+  display: grid;
+  grid-template-columns: 4rem 1fr 4rem;
+  grid-template-areas: "b t .";
+  align-items: center;
+  position: sticky;
+  top: 0;
+  padding: 1rem;
+  background-color: var(--header-background);
+  -webkit-backdrop-filter: blur(10px) saturate(150%);
+  backdrop-filter: blur(10px) saturate(150%);
+  box-shadow: 0 1px 8px rgba(0,0,0,0.1);
+}
+h1 {
+  grid-area: t;
+  margin: 0;
+  text-align: center;
+  font-size: 1rem;
+  font-weight: 700;
+  color: var(--title-color);
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+}
+main {
+  padding: 1rem;
+  width: fit-content;
+  max-width: 100%;
+  margin: 0 auto;
+  overflow-x: auto;
+}
+.error {
+  margin: 1rem auto;
+  width: fit-content;
+  max-width: 90%;
+  padding: .75rem 1rem;
+  text-align: center;
+  background-color: rgba(235, 87, 87, 0.15);
+  border-radius: .5rem;
+}
+@media (prefers-color-scheme: dark) {
+  .error {
+    color: #EB5757;
+  }
+}
+.icon {
+  display: inline-block;
+  vertical-align: middle;
+  width: 1rem;
+  height: 1rem;
+  fill: var(--accent-color);
+}
+.source {
+  padding: .75rem 0;
+  font-size: 1rem;
+  color: var(--title-color);
+}
+.list {
+  width: 90ch;
+}
+.list a {
+  display: block;
+  padding: 1rem 0;
+  font-size: .9375rem;
+  font-weight: 500;
+  color: var(--text-color);
+  border-top: .5px solid var(--border-color);
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+}
+.list .icon {
+  margin-right: .5rem;
+}

